ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/route_maps.py

```diff
@@ -97,13 +97,13 @@ def route_maps(self) -> list | None:
             # RM-BGP-UNDERLAY-PEERS-IN
             sequence_numbers = [
                 {
-                    "sequence": 40,
+                    "sequence": 30,
                     "type": "permit",
-                    "description": "Mark prefixes originated from the LAN",
+                    "description": "Set SOO on prefixes from the site.",
                     "set": [f"extcommunity soo {self.shared_utils.evpn_soo} additive"],
```

Comment:
These are breaking changes. Can we revert and submit an issue for 5.0 to adjust this?

Reply:
so WAN HA was preview - I could make these back to what they were (40 and same description) and it would not be breaking - let me do this.

Reply:
Reverted